common/changes/@rushstack/heft-config-file/user-danade-HeftJestPreset_2023-04-24-22-34.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft-config-file",
+      "comment": "The custom resolver method now accepts an options parameter. This parameter includes all previously provided information and now includes the partially-resolved configuration file.",
```

Comment:
This is a breaking API change. Can you note that in the change description?

I wouldn't make this a major bump, given that it's a `@beta` API.